packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -176,10 +176,10 @@ function ListItem< Item >( {
 	}, [ actions, item ] );
 
 	const renderedMediaField = mediaField?.render ? (
-		<mediaField.render item={ item } />
-	) : (
-		<div className="dataviews-view-list__media-placeholder"></div>
-	);
+		<div className="dataviews-view-list__media-wrapper">
```

Comment:
This removes the placeholder in all `lists`, right?

I'm wondering why we needed an empty placeholder before though, because it would only be rendered if we hadn't provided `mediaField` and wouldn't check if the result of the `render`. 🤔

Reply:
Yeah, it removes it when the consumer doesn't provide a `mediaField`, while before we'd render a placeholder.